sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -817,4 +818,11 @@ public static RetryContext getRetryContext(CosmosDiagnostics cosmosDiagnostics)
             return null;
         }
     }
+
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
+    public static <TContext> ConcurrentMap<String, PartitionScopeThresholds<TContext>> getPartitionScopeThresholds(
```

Comment:
ideally we should add more methods in this file, as this technically will be considered public API.

please see this as alternative for how to do cross package referencing:
https://github.com/Azure/azure-sdk-for-java/blob/83b6be3498cb5b6b5912688eb6d2efe0231fe6f0/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ImplementationBridgeHelpers.java

Reply:
ACK - Fixed